File: Jessica/9.Palindrome Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x<0:
            return False
        strx=str(x)
        length=len(strx)
        str1=strx[0:int(length/2)]
        str2=strx[:-int(length/2)-1:-1]
        if str1 == str2:
            return True
        return False
        
     

File: Jessica/9.Palindrome Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
          if (x<0)
            return false;
        else if(x<10 && x>=0)
            return true;
        else{
            int reverse = 0;
            int y=x;
            while(y>0){
                reverse=(reverse*10)+(y%10);
                
                y=y/10;
            }
            if(reverse == x)
                return true;
            else
                return false;
                  
                
            
             
        }
    }
};